tc/core/polyhedral/cuda/mapped_scop.cc

```diff
@@ -517,6 +499,16 @@ isl::multi_aff constructThreadToWarp(
 }
 } // namespace
 
+isl::multi_union_pw_aff MappedScop::threadMappingSchedule(
+    const detail::ScheduleTree* tree) const {
+  std::vector<mapping::MappingId> ids;
+  for (int i = 0; i < numThreads.view.size(); ++i) {
```

Comment:
warning: comparison between signed and unsigned integer expressions

Reply:
We seem to compile with `-Wno-sign-compare`